tests/fast-integration/logging/client.js

```diff
@@ -1,7 +1,6 @@
-const axios = require('axios');
 
 const {
-    kubectlPortForward,
+    kubectlPortForward, getResponse,
```

Comment:
just for having consistent style, can you please place `getResponse` in a new line?